judging/scripts/bootstrap.sh

```diff
@@ -20,7 +27,7 @@ echo "Running bootstrap at $(date)"
 repo_name="covid-xprize-robotasks"
 
 # Use this branch from Github
-branch="main"
+branch="phase2"
```

Comment:
Temporary for testing -- we'll switch it back to `main` for production.

Reply:
How are we going to remember to switch it back? Can you switch it back now and edit it locally for testing?

Reply:
Yes, actually this particular change doesn't need to be committed since this file is deployed manually (other "dummy" changes do need to be committed since the script logic picks everything up from Github)